### PR TITLE
Adding extra details slot, adding new bolding rule

### PR DESCRIPTION
**Pull request by liechtym**
- Added slot for adding extra details to a person, especially so users can add "view relationship" to fs-person
- Made it so if an anchor element is passed in to wrap the person's name, bolding is removed (styleguide decision)
- Added tests, demos, and docs for the above.

### CODE REVIEW

**Comment by timothydm42** [comment]
What do you think about adding a little section about this new slot to the READme?

**Comment by liechtym** [comment]
@timothydm42 Good call, I'll do that

**Comment by liechtym** [comment]
@timothydm42 I added a section to the readme. Thanks for the suggestion!

File: fs-person.html [hunk 1]
```diff
@@ -80,6 +84,10 @@
   line-height: var(--fs-line-height-small, 1.35rem);
 }
 
+.fs-person__details.extra-details {
```
Comment (timothydm42) [inline fs-person.html]: PR looks great. my only question is on this selector. I'm not clear on what it does so I need you to educate me :) let's talk about it when you get a chance

**Pull request merged**